designsafe/static/scripts/data-depot/components/published/published.component.js

```diff
@@ -117,17 +113,8 @@ export class PublishedDataCtrl {
         };
     }
 
-    getFileObjs(evt) {
-        evt.files = _.map(evt.fileObjs, (f) => {
-            f.system = 'designsafe.storage.published';
-            f.path = this.browser.publication.projectId + f.path;
-            f.permissions = 'READ';
-            return this.FileListing.init(f, _apiParams);
-        });
-    }
-
     getTitle() {
-        this.$window.document.getElementsByName('citation_author_institution')[0].content = 'dingus'
+        this.$window.document.getElementsByName('citation_author_institution')[0].content = 'dingus';
```

Comment:
`'dingus'`? What's this for? :)

Reply:
Uh that's a good question lol. I think I just put a semicolon there because of my linter. I didn't check to see what this actually does.